ts/tests/blueprints/ember-cli-typescript-test.ts

```diff
@@ -136,9 +138,7 @@ describe('Acceptance: ember-cli-typescript generator', function() {
 
         const globalTypes = file('types/global.d.ts');
         expect(globalTypes).to.exist;
-        expect(globalTypes)
-          .to.include("declare module 'my-addon/templates/*'")
-          .to.include(`
+        expect(globalTypes).to.include("declare module 'my-addon/templates/*'").to.include(`
```

Comment:
IMHO, this is harder to read than what we had. What rule is forcing this?

Reply:
This is just Prettier's default behavior with maximizing use of line length. This kind of thing is why Prettier strongly recommends the maximum length of 80; I had set it to 100 to match other projects I was working on back when I included a config a long time ago, but I have no objection to doing it at 80 instead. (I actually considered doing that here but opted to leave the settings as they were and just actually enable it.)

Reply:
Looks like it's the multiline template literal that's causing this. If it were not for that, prettier will do [this](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHOYYAoDmANhAEYCGhAKgJ5YDOAlAHQwRMCWUYhArgCZxcAHRADuZAE5wABAFsIfHoRkByWdQC0ZPn2gB6eLIyEy8OnoBUKkc1YcuvAbgAGAKx50Y06DMKc4zgwA3CAANCAQGDDs0HTIoJISEADuAAqSCHEoZABuEOx8YSAkEmRgANZwMADKGGWc+MgwEjxw4QAWMLKEAOrt7GZ1YHDVmQPsOQPUyOB0ceGcdHASMKml+LJkyABmFEvhrnRoAEKlFVXVZLJwADL+O3ttIIdo1Q3KAIo8EPAPhPsgOoSJYSGbkEjUYhQIoYCScGA9AowdrIACMAAZ0eFYRAlj1ShgZrC4CCcnAigBHb7wNaRLIgMh0DRQOBwASFcJSKnsKRrMgbLZIXb-J5LWTsJotUXvOBfH7kpDNVrhGBkEiIvjI5AAJhVpXYfig+AAwhBZJsZhRCEUPHBKGqssKAYyYH8AbowG6nhAeDAML7tV6AL5BoA).

Reply:
Turns out if you manually wrap the string to start on the next line, it does what we would all call the right thing.

Reply:
Ah, very nice. Can you manually wrap this one?

Reply:
Yep, and in the latest change (1e6a85b) *is* wrapped. 🎉

Reply:
Wait, it's still not wrapped on this line: https://github.com/typed-ember/ember-cli-typescript/pull/678/files#diff-1411d46f2ad90f63434bf68ee3f6fcdeR141

Reply:
That's the old commit from when you made the original comment, I believe.